python_app/utils.py

```
@@ -0,0 +1,44 @@
+
+
+def parse_sensor_payload(raw: Any) -> Any:
+    """Parse raw sensor payload into structured data."""
+    result: Any = {
+        "temperature": raw.get("temp"),
+        "pressure": raw.get("pres"),
+    }
+    return result
+
+
+def validate_beacon_id(beacon_id: Any) -> Any:
```

Ruff (0.15.6)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `beacon_id`

(ANN401)